packages/forms-web-app/src/views/appellant-submission/task-list.njk

```diff
@@ -3,7 +3,7 @@
 {%- from "components/task-list/macro.njk" import mojTaskList -%}
 
 {% block pageTitle %}
-  Appeal a planning decision - Appeal a planning decision - GOV.UK
+  Appeal a planning decision - GOV.UK
```

Comment:
this is a different page from the one mentioned in the bug - presumably it requires the same text change, but possibly worth checking with design